src/Modal/Modal.jsx

```diff
@@ -62,11 +119,19 @@ Modal.propTypes = {
     PropTypes.node
   ]).isRequired,
   onClose: PropTypes.func,
+  onCancel: PropTypes.func,
+  onSubmit: PropTypes.func,
+  cancelText: PropTypes.string,
+  submitText: PropTypes.string,
   role: PropTypes.string,
   ariaLabel: PropTypes.string
 };
 Modal.defaultProps = {
-  onClose: () => {},
+  onClose: null,
+  onCancel: null,
+  onSubmit: null,
+  cancelText: "",
```

Comment:
cancel and submit can have default text

Reply:
then this text wont be translatable. we dont translate from vivy-components

Reply:
yes, it should be the responsibility of service to add translations, but still btns should be usable from the very beginning

Reply:
yeah agree, why dont we make it that when the user has passed the prop onSubmit but not any text the propValidation would complain that a text should be added, also an error will be shown on the console? This way we restrict the user to always have text passed in the button when the usecase is to have Sumbit button shown.